snapcraft/cli/lifecycle.py

```diff
@@ -14,32 +14,23 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import subprocess
 import logging
 import typing
-import sys
 import os
-from time import sleep
 
 import click
 
 from . import echo
 from . import env
 from ._options import add_build_options, get_project
 from snapcraft.internal import (
-    errors,
     build_providers,
     deprecations,
     lifecycle,
     project_loader,
     steps,
-    repo,
 )
-from snapcraft.project._sanity_checks import (
-    conduct_project_sanity_check,
-    conduct_build_environment_sanity_check,
-)
-from snapcraft.project.errors import MultipassMissingInstallableError
+from snapcraft.project._sanity_checks import conduct_project_sanity_check
```

Comment:
I believe this would be cleaner if conduct_project_sanity_check is exported by snapcraft.project instead of accessing project._sanity_checks directly from a different module.

Reply:
the option is to either keep the module private or make the method private (with _).
I am not sure which I prefer, but considering that this is not part of the PR, I think we should leave as is :wink: